config/examples/transparent.yml

```diff
@@ -0,0 +1,19 @@
+log_debug: true
+
+server:
+  http:
+      listen_addr: ":9090"
+      allowed_networks: ["127.0.0.0/24"]
+
+users:
+  - name: "transparent_user"
+    to_cluster: "default_with_transparent"
+    to_user: "transparent_user"
```

Comment:
allow_transparent_cache: true ?

Reply:
There is no cache in this config, so the value does not matter, while it is false by default.
The use case where a transparent user exists and the cache enabled is covered by bad.transparent_user_with_cache.yml and correspondent test